irc/Cargo.toml

```diff
@@ -10,6 +10,7 @@ futures = "0.3.28"
 thiserror = "1.0.30"
 tokio = { version = "1.29", features = ["net", "full"] }
 tokio-rustls = { version = "0.26.0", default-features = false, features = ["tls12", "ring"] }
+tokio-socks = "0.5.1"
```

Comment:
FYI this crate is unmaintained.

We may want to use the `fast_socks5` crate, it seems maintained and built on top of tokio

Reply:
I just updated the `TcpStream` preparation to use the `fast_socks5` crate.